js/src/Ice/InputStream.d.ts

```diff
@@ -0,0 +1,310 @@
+//
+// Copyright (c) ZeroC, Inc. All rights reserved.
+//
+
+declare module "ice" {
+    namespace Ice {
+        /**
+         * The InputStream class provides methods for decoding Slice encoded data from a stream of bytes.
+         */
+        class InputStream {
+            constructor();
```

Comment:
Should we add doc-comment to the constructors?

Reply:
Yes added doc comments to constructors.